hermes-mock/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'groovy'
+    id 'java'
+}
+
+dependencies {
+    compile project(':hermes-test-helper')
```

Comment:
There are an awful lot of classes there, we cannot pull them into our user's classpath.